afqbrowser/site/client/js/save-settings.js

```diff
@@ -168,3 +168,15 @@ afqb.table.restoreRowSelection = function () {
         }
     });
 };
+
+$('#launch-binder').click(function (e) {
+    e.preventDefault();
+    // Parse the URL
+    var uri = new URI(location.href);
+    var user = uri.hostname();
+    var repo = uri.directory();
+    var binderUrl = '//mybinder.org/v2/gh/' + user + '/' + repo + '/gh-pages?filepath=index.ipynb';
```

Comment:
Also note that this uses a protocol-relative URL so coping the URL from the console and pasting into a new tab won't work. It will think it's a file location so you'll have to add "http:" or "https:". But `window.open()` should know how to handle it and will just use whatever protocol the user is currently using (I think).